<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>427</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>665</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="rightMargin">
        <number>15</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="formatsListLabel">
         <property name="text">
          <string>formatsList</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="formatsListWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <item row="1" column="1">
        <widget class="QPushButton" name="newProjBtn">
         <property name="text">
          <string>newProject</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="opnProjBtn">
         <property name="text">
          <string>openProject</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="convProjBtn">
         <property name="text">
          <string>convertProject</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="settBtn">
         <property name="text">
          <string>settings</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="formatInfoLabel">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>170</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>600</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec magna mauris, sodales a faucibus quis, sodales tincidunt ligula. Duis hendrerit aliquet tortor, eu fringilla odio feugiat a. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec magna mauris, sodales a faucibus quis, sodales tincidunt ligula. Duis hendrerit aliquet tortor, eu fringilla odio feugiat a.</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="newTmBtn">
         <property name="text">
          <string>newTm</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="convTmBtn">
         <property name="text">
          <string>convertTm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
